obs-studio-server/source/util-crashmanager.cpp

```diff
@@ -1046,7 +1046,7 @@ void util::CrashManager::SaveToAppStateFile()
 	if (current_status.size() == 0) 
 		return;
 	
-	const std::string freez_flag = "ipc_freez";
+	const std::string freez_flag = "obs_crash";
```

Comment:
I don't understand the logic behind the naming here, could you explain why the freeze flag is considered as an obs crash?

Reply:
Good point. 
Changed var name to not include mention of exact value.